src/components/products.js

```diff
@@ -0,0 +1,27 @@
+import products from '../data/products';
+
+function Product(product) {
+  const { image: src, name, url, rating, price, description } = product;
+  return (
+    <div className='product'>
+      <img src={src} alt={description} />
+      <a href={url}>{name}</a>
+      <div className='product__footer'>
+        <div className='product__rating'>
+          <span>{rating}</span>
+        </div>
+        <span className='product__price'>{price}</span>
+      </div>
+    </div>
+  );
+}
+
+function display(array) {
+  array.map(({ id, ...rest }) => <Product key={id} product={rest} />);

```

Comment:
hoặc
```suggestion
  array.map(({ id, ...rest }) => <Product key={id} {...rest} />);
```
lúc này component Product sẽ là
`function Product(image: src, name, url, rating, price, description }) {}`